fog/view/server/src/db_fetcher.rs

```diff
@@ -351,6 +363,8 @@ impl<DB: RecoveryDb + Clone + Send + Sync + 'static> DbFetcherThread<DB> {
                     );
                     // We might have more work to do, we aren't sure because of the error
                     may_have_more_work = true;
+                    // Let's back off for one interval when there is an error
+                    sleep(DB_POLL_INTERNAL);
```

Comment:
Won't this get achieved simply by not setting `may_have_more_work = true`?

Reply:
IIUC that would make the server mark itself ready when it can't load any data

Reply:
Oh yeah, you're right.